Lib/subprocess.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, *, dwFlags=0, hStdInput=None, hStdOutput=None,
             self.hStdOutput = hStdOutput
```

Comment:
For convenience, do you think the handles in `handle_list` should be automatically duplicated as inheritable via `_make_inheritable`? The problem with that is managing the lifetime of the `Handle` instances. It would have to make a private copy of `STARTUPINFO`, which maybe it should be doing anyway.